src/screens/BiometricScreen/BiometricScreen.tsx

```diff
@@ -0,0 +1,104 @@
+import {useCallback} from 'react';
+import {useNavigation} from '@react-navigation/native';
+import {NativeStackNavigationProp} from '@react-navigation/native-stack';
+import {Alert, StyleSheet, Text, View} from 'react-native';
+import ReactNativeBiometrics, {BiometryTypes} from 'react-native-biometrics';
+import {useActions} from 'hooks/useActions';
+import {StyledButton} from 'components';
+import {useThemedStyles, width} from 'styles/commonStyles';
+import {ScreenNames, StackParamList} from 'types/navigation';
+
+export default function BiometricScreen() {
+  const navigation = useNavigation<NativeStackNavigationProp<StackParamList>>();
+  const {onSetBiometrics} = useActions();
+  const themedStyles = useThemedStyles();
+
+  const showAlert = useCallback(
+    (title: string, message: string, biometryType: string) => {
+      Alert.alert(title, message, [
+        {
+          text: 'Yes please',
+          onPress: async () => {
+            try {
+              // Create & save publicKey for the real app
+              onSetBiometrics();
+              navigation.navigate(ScreenNames.Home);
+              Alert.alert(
+                'Success!',
+                `${biometryType} authentication enabled successfully!`,
+              );
+            } catch (error) {
+              console.error('Keychain error:', error);
+              Alert.alert(
+                'Error',
+                'An error occurred while enabling biometrics.',
+              );
+            }
+          },
+        },
+        {text: 'Cancel', style: 'cancel'},
+      ]);
+    },
+    [onSetBiometrics],
+  );
+
+  const enableBiometricAuth = useCallback(async () => {
+    const rnBiometrics = new ReactNativeBiometrics();
+
+    try {
+      const resultObject = await rnBiometrics.isSensorAvailable();
+      const {available, biometryType} = resultObject;
+
+      if (available) {
```

Comment:
let's have an earlier return:
`if (! available ) {showAlert(); return;}`
and also let's re-write if-else below -> switch/case